recipes/opengl-headers/all/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conan import ConanFile
+from conan.tools.files import copy, get, load, save
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class OpenGLHeadersConan(ConanFile):
+    name = "opengl-headers"
```

Comment:
Why not `openg-registry`?

Reply:
Done here: d03b2777e

Reply:
the fact that the headers are stored in a website called registry is quite irrelevant to the naming of the recipe, which actually packages opengl's header files